include/stlink/tools/flash.h

```diff
@@ -20,9 +22,10 @@ struct flash_opts
     int reset;
     int log_level;
     enum flash_format format;
+    uint32_t flash_size;	/* --flash=n[k][m] */
```

Comment:
you could better use size_t here, as we don't map it back to the microcontroller